src/index.html

```diff
@@ -1,13 +1,468 @@
+              </li>
+            </ul>
+          </nav>
+
+          <section
+            class="our-story"
+            id="our-story"
+            data-aos="fade-up"
+          >
+            <div class="our-story__card">
+              <img
+                src="./icons/measure.svg"
+                alt="compass icon"
+                class="our-story__icon"
+              >
+              <h3 class="our-story__title">Futuristic Design</h3>
+              <p class="our-story__text">
+                To give Luna a truly flawless look, we specifically picked aircraft grade aluminum
+                as its material and adopted both three-dimensional stretch-bending technology and a high precision cold
+                forging technique.
+              </p>
+            </div>
+
+            <div class="our-story__card our-story__card-tablet">
+              <img
+                src="./icons/sound.svg"
+                alt="speaker icon"
+                class="our-story__icon"
+              >
+              <h3 class="our-story__title">Tweeter Speaker System</h3>
+              <p class="our-story__text">
+                To deliver a more layered sound performance better than a sole full-range speaker, our team equipped Luna
+                with one more tweeter speaker responsible for high-frequency sound independently.
+              </p>
+            </div>
+
+            <div class="our-story__card our-story__card-tablet">
+              <img
+                src="./icons/wifi.svg"
+                alt="wi-fi icon"
+                class="our-story__icon"
+              >
+              <h3 class="our-story__title">Futuristic Design</h3>
+              <p class="our-story__text">
+                Luna is natively compatible with your home Wi-Fi. Set up multiple speakers in different rooms to expand your
+                music experience into the entire house.
+              </p>
+            </div>
+
+            <div class="our-story__card our-story__card-tablet">
+              <img
+                src="./icons/sun.svg"
+                alt="compass icon"
+                class="our-story__icon"
+              >
+              <h3 class="our-story__title">Intuitive Lighting System</h3>
+              <p class="our-story__text">
+                An intuitive user interface allows you to adjust the hue and saturation of color for lighting that fits any
+                mood and situation.
+              </p>
+            </div>
+          </section>
+
+          <section data-aos="fade-up"
+          >
+            <img
+              class="luna-img"
+              src="./images/meetLuna.png"
+              alt="luna pictures"
+            >
+          </section>
+        </div>
+      </div>
+
+      <section class="about-us" id="about-us">
+        <article class="about-us__article" data-aos="fade-up">
+          <div class="about-us__content about-us__content-1">
```

Comment:
`about-us__content` is an element of block `about-us__article`, not of block `about-us`,
I think you can rename `about-us__article` to `article`, because the card block is self-sufficient and can be in other sections of pages